.github/workflows/lint.yml

```diff
@@ -23,12 +23,6 @@ jobs:
 
       - run: yarn --frozen-lockfile
 
-      # v1 is broken, the given sha was the latest master (but we dont want to track master)
-      - uses: tinovyatkin/action-eslint@8b32772f075413f9192f43641e03d9bd0abae621
-        with:
-          repo-token: ${{secrets.GITHUB_TOKEN}}
-          check-name: eslint # this is the job name from above 👆
```

Comment:
It's nice to get the annotations inline, so perhaps we can do `continue-on-error`:

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error

And then we can rely on `yarn lint` to pass/fail the check (remove the `if: always` from that check)